app/views/layouts/govuk_template2.scala.html

```diff
@@ -85,6 +101,15 @@
   </head>
 
   <body class="@bodyClasses.getOrElse("")">
+    <!-- Google Tag Manager (noscript) -->
+    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-@{current.configuration.getString("analytics.ga.GMT").get}"
+                      height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+    <!-- End Google Tag Manager (noscript) -->
+    @if(needsTimeoutWarning){
```

Comment:
Why we just don't embed dome javascript code instant of form inputs?

Reply:
that's pretty much what Andrew told me to paste, so I rather do as he says

Reply:
https://jira.tools.tax.service.gov.uk/browse/VOA-121
>Copy the following snippet and paste it immediately after the opening <body> tag on every page of RALD, Contact Finder, Digital Contact service, ROFR and BARS.